src/x86/sse41.rs

```diff
@@ -10,10 +11,42 @@ pub fn _mm_blendv_epi8(
     unsafe { pblendvb(a, b, mask) }
 }
 
+/// Returns the dot product of two f64x2 vectors
+/// Imm8[1:0] is the broadcast mask, and the imm8[5:4] is the condition mask
+/// If a condition mask bit is zero, the corresponding multiplication is replaced by a value of 0.0
+/// If a broadcast mask bit is one, the result of the dot product will be stored in the return value component,
+/// if it is zero then the return component will be zero.
```

Comment:
The content here looks good, but could you please reformat the docs so that there is proper punctuation and so that they are wrapped to 79 columns inclusive? Here's how I'd format it with some wording tweaks:

```rust
/// Returns the dot product of two f64x2 vectors.
///
/// `imm8[1:0]` is the broadcast mask, and `imm8[5:4]` is the condition mask.
/// If a condition mask bit is zero, the corresponding multiplication is
/// replaced by a value of `0.0`. If a broadcast mask bit is one, the result of
/// the dot product will be stored in the return value component. Otherwise if
/// the broadcast mask bit is zero then the return component will be zero.
```

Reply:
It looks better this way, if you don't mind, I will just copy and paste it.